src/Executor/ExecutionResult.php

```diff
@@ -66,14 +66,14 @@ class ExecutionResult implements \JsonSerializable
      *
      * @phpstan-var ErrorFormatter|null
      */
-    private $errorFormatter = null;
+    private $errorFormatter;
```

Comment:
Not sure if this is correct? I don't see it initialized in constructor

Reply:
It is an untyped property, so it does not need initialization.